Where_Is_My_Wife/Assets/_Main/Scripts/Stage/Earth/TirePunchingDistanceIndicator.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class TirePunchingDistanceIndicator : MonoBehaviour
```

Comment:
Erase empty methods and add a comment with a TODO explaining why this is going to be used.